xrootdfs/fs.py

```diff
@@ -56,11 +62,26 @@ def __init__(self, url, query=None, timeout=0):
         if not is_valid_path(base_path):
             raise InvalidPathError(path=base_path)
 
+        if queryargs:
```

Comment:
Could add a comment here briefly explaining what's going on :P 
(e.g. "extract query args as (x, v) dict (where x is...), raise error if...)"